src/LSF.jl

```diff
@@ -0,0 +1,22 @@
+normal_pdf(Δ, σ) = exp(-0.5*Δ^2 / σ^2) / √(2π) / σ
+
+"""
+Returns the matrix which transforms a spectrum to its LSF-convolved version for spectral resolution 
```

Comment:
could you maybe just spell out "line spread function"? Other than that, this seems great